_includes/cwl/15-staging/linkfile.cwl

```diff
@@ -8,8 +8,7 @@ hints:
 baseCommand: javac
 
 requirements:
-  - class: InlineJavascriptRequirement
-  - class: InitialWorkDirRequirement
+  InitialWorkDirRequirement:
```

Comment:
This syntax is not valid in v1.0.2 because the spec says that `requirements` should be an array.

Reply:
In the spec itself a mapping using the value of the `class` field as the key is allowed: https://github.com/common-workflow-language/common-workflow-language/blob/master/v1.0/Process.yml#L811 but I see that it isn't rendering in the HTML version: http://www.commonwl.org/v1.0/CommandLineTool.html#CommandLineTool I also reviewed all 10 captures of this page in archive.org across 2016 & 2017 and the same rendering bug is present.

@tetron Any ideas?

Reply:
I've opend an issue about this at https://github.com/common-workflow-language/schema_salad/issues/167 and I think it is rather important to fix.

Reply:
@mr-c I got it. Thanks for the information!